linux_os/guide/system/accounts/accounts-pam/display_login_attempts/rule.yml

```diff
@@ -4,11 +4,18 @@ title: 'Ensure PAM Displays Last Logon/Access Notification'
 
 description: |-
     To configure the system to notify users of last logon/access
-    using <tt>pam_lastlog</tt>, add or correct the <tt>pam_lastlog</tt> settings in
+    using <tt>pam_lastlog</tt>, add or correct the <tt>pam_lastlog</tt>
+    settings in
+    {{% if product == "sle12" %}}
+    <tt>/etc/pam.d/login</tt> to read as follows:
```

Comment:
Thanks for pointing this out. When comparing SLES12 and RHEL7 stig https://vaulted.io/library/disa-stigs-srgs/sles_12_security_technical_implementation_guide/V-217140 and https://vaulted.io/library/disa-stigs-srgs/red_hat_enterprise_linux_7_security_technical_implementation_guide/V-204605 they differ only in path `/etc/pam.d/login` vs. `/etc/pam.d/postlogin`. Otherwise they're the same. So I think in future we will update the RHEL branch in a similar manner.

Reply:
Thanks